python/setup.py

```diff
@@ -208,12 +208,18 @@ def build_extension(self, ext: CMakeExtension) -> None:
     # See issue #222: https://github.com/SINTEF/dlite/issues/222
     # extras_require={"all": extra_requirements},
     packages=["dlite"],
+    scripts=[
+        #str(SOURCE_DIR / "bindings" / "python" / "scripts" / "dlite-validate"),
+    ],
     package_data={
         "dlite": [
             dlite_compiled_ext,
             dlite_compiled_dll_suffix,
             str(Path(".") / "share" / "dlite" / "storage-plugins" /
                 dlite_compiled_dll_suffix),
+            str(Path(".") / "bin" / "dlite-getuuid"),
+            str(Path(".") / "bin" / "dlite-codegen"),
+            str(Path(".") / "bin" / "dlite-env"),
```

Comment:
Could you not just do a single line with a glob expression?
E.g.,: `str(Path(".").resolve() / "bin" / "*")` or something similar.

Reply:
No, that is dangerous and might pick up things we don't want to install.